### PR TITLE
Fix integration tests

### PR DESCRIPTION
**Pull request by rupertbates**
## Why are you doing this?

A few of the integration tests had become broken due to recent changes, this PR fixes those tests or removes them where they are no longer needed. I have also added a pre-push git hook to make sure that the tests stay green in future.

[**Trello Card**](https://trello.com/c/sYEQ9kbt/2084-post-deploy-test-for-landing-pages)

### CODE REVIEW

File: scripts/pre-push [hunk 1]
```diff
@@ -0,0 +1,10 @@
+echo "Running integration tests before push"
+sbt it:test
```
Comment (johnduffell) [inline scripts/pre-push]: not a bad idea!!

File: src/main/scala/com/gu/ophan/AcquisitionService.scala [hunk 2]
```diff
@@ -2,7 +2,7 @@ package com.gu.ophan
 
 import com.gu.okhttp.RequestRunners
 
-object OphanService {
+object AcquisitionService {
```
Comment (johnduffell) [inline src/main/scala/com/gu/ophan/AcquisitionService.scala]: yes that did confuse me a bit before as well, good improvement

File: src/main/scala/com/gu/zuora/model/response/Responses.scala [hunk 3]
```diff
@@ -77,20 +77,13 @@ object SubscribeResponseAccount {
...
-  totalTcv: Float,
-  subscriptionId: String,
-  totalMrr: Float,
-  paymentTransactionNumber: String,
-  accountId: String,
-  gatewayResponseCode: String,
-  invoiceNumber: String,
-  invoiceId: String,
-  success: Boolean
+    accountNumber: String,
```
Comment (johnduffell) [inline src/main/scala/com/gu/zuora/model/response/Responses.scala]: this always happens to me as well for some reason, but it was actually correct before!

**Comment by johnduffell** [comment]
looks great, thanks for going through all this and nice idea making it a pre push.  👍

File: src/main/scala/com/gu/zuora/model/response/Responses.scala (hunk 3 above)
Comment (rupertbates) [reply src/main/scala/com/gu/zuora/model/response/Responses.scala]: It's Scalariform messing about, I've reverted this change

**Pull request merged**